htdocs/core/lib/signature.lib.php

```diff
@@ -77,7 +77,7 @@ function getOnlineSignatureUrl($mode, $type, $ref = '', $localorexternal = 1)
 	$securekeyseed = '';
 
 	if ($type == 'proposal') {
-		$securekeyseed = $conf->global->PROPOSAL_ONLINE_SIGNATURE_SECURITY_TOKEN;
+		$securekeyseed = isset($conf->global->PROPOSAL_ONLINE_SIGNATURE_SECURITY_TOKEN) ? $conf->global->PROPOSAL_ONLINE_SIGNATURE_SECURITY_TOKEN : '';
```

Comment:
Note that with v16, you can use 
$securekeyseed = getDolGlobalConst('PROPOSAL_ONLINE_SIGNATURE_SECURITY_TOKEN')
that is better.

Reply:
where is the function?